.env.development.local

```diff
@@ -6,6 +6,7 @@ REACT_APP_SUPABASE_URL=http://localhost:5431
 REACT_APP_SUPABASE_ANON_KEY=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs
 REACT_APP_GATEWAY_AUTH_TOKEN_URL=http://localhost:5431/rest/v1/rpc/gateway_auth_token
 
+REACT_APP_SHOW_EMAIL_LOGIN=true
```

Comment:
We need to keep this on for local dev. Makes setting up locals easier and is also the best option to login when testing locally through a image.